ipaplatform/fedora/constants.py

```diff
@@ -16,6 +16,8 @@
 # Fedora 30 and later use /etc/nfs.conf
 # Fedora 29 has both
 HAS_NFS_CONF = osinfo.version_number >= (30,)
+# BIND employs 'pkcs11' OpenSSL engine instead of native PKCS11
+HAS_BIND_OPENSSL_ENGINE = osinfo.version_number >= (31,)
```

Comment:
You can simplify the extra checks and assume that Fedora has OpenSSL engien support. Fedora 30 is no longer supported.

Reply:
ok, done.